### PR TITLE
Normalize de locales file

### PR DESCRIPTION
**Pull request by coorasse**
**This PR normalises German locales file.**

I followed the README and decided to update these locales since there are some missing. During this process, the task `i18n-tasks add-missing` normalised my file.

I decided to split my work into different PRs and avoid mixing the addition of new translations with the normalisation of the file.

Since the rubocop configuration is set to a maximum length for the line of 140 characters, I configured `i18n-tasks` to behave the same during the normalization.

I then simply run `i18n-tasks normalize` on the German translations. 

I think this task should be part of the CI, in order to keep our translations in order (pun intended).

If you also think this is a good thing, I will open a PR to introduce `i18n-tasks check-normalized` in the CI and fix all the other locales.

TL;DR;
This PR does not add any translation but just commit the result of `i18n-tasks normalize --locale de`

### CODE REVIEW

**Comment by kennyadsl** [comment]
> If you also think this is a good thing, I will open a PR to introduce i18n-tasks check-normalized in the CI and fix all the other locales.

I think it's a good idea. 

We've started something similar in core as well for `en`. Some related PRs to check:

- https://github.com/solidusio/solidus/pull/2963
- https://github.com/solidusio/solidus/pull/2988
- https://github.com/solidusio/solidus/pull/2972

Feel free to propose the same solution for Solidus core. 🙂
